src/modules/enrollment/components/dashboardPages/EnrollmentOverview.tsx

```diff
@@ -59,7 +59,7 @@ const EnrollmentOverview = () => {
     <>
       <PageTitle title='Enrollment Overview' />
       <Grid container spacing={4}>
-        <Grid item xs={7}>
+        <Grid item md={8} xs={12}>
```

Comment:
The grid item's width was previously 7 instead of 8 here, meaning that the two side-by-side uneven-width components didn't fill the whole 12 available columns. I switched it to 8 on the idea that this was probably a mistake, but let me know if there was actually an important reason for it!

Reply:
This looks good to me, thanks!